scripts/talos/getChangelogHead/index.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const getPackagePath = require('../../utilities/getPackagePath');
+const readFile = require('../../utilities/readFile');
+
+const getChangelogHead = publishedPackage => {
+  const packageName = publishedPackage.split(' ')[0];
+  const packageVersions = publishedPackage
+    .replace(packageName, '')
+    .split('→')
+    .map(version =>
+      version
+        .trim()
+        .replace('^', '')
+        .replace('~', ''),
+    );
+
+  const packagePath = getPackagePath(packageName);
+  const changelogLines = readFile(path.join(packagePath, 'CHANGELOG.md')).split(
+    '\n',
+  );
+
+  const tableHeadRegex = /\|\s+?Version\s+?|\s+?Description\s+?\|/;
+  const tableHeadIndex = changelogLines.findIndex(line =>
+    line.match(tableHeadRegex),
+  );
+  const firstVersionIndex = changelogLines.findIndex(line =>
+    line.includes(packageVersions[0]),
```

Comment:
Not tested it, but could this get confused with `1.2.1` and `1.2.10`? So we miss changes `1.2.2` => `1.2.9`? May need to ensure its followed and lead by a non-number character? (likely a space of `|`, accounting for broken formatting :P)

Reply:
True, I think a space suffix should fix it.